### PR TITLE
Add profile pic, and PPP.

### PR DESCRIPTION
**Pull request by josephhhhhhhhh**
For #214 .

### CODE REVIEW

File: docs/team/joseph.md [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
+    at which point, each question's options will be printed under the question, with the words "[Correct Answer]" next to the option whose attribute `isCorrectAnswer` is true.
+    The user may also delete a bookmark by typing "bookmark delete <index number of the question>", at which point the question indicated will be deleted.
+    - Justification: This feature is important in terms of using the product, as from time to time questions that are hard to figure out may get tested, and the user
+    would want to have a way to store this question for perusal later. By consolidating all these bookmarks in one place, the user is capable of referring to all the questions that they have had doubts about
+    in one central location. This makes revising an easier process as well.
+    - Highlights: This command requires interaction with the `Ui`, `Question` and `Option` classes in order to be able to store and display the questions effectively. It requires altering the code of some pre-existing
```
Comment (anqi20) [inline docs/team/joseph.md]: "It requires altering **of** the code"?

**Review by anqi20 — APPROVED** [review]
Other than that, I think it looks good!

File: docs/AboutUs.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
-![](https://via.placeholder.com/100.png?text=Photo) | Joel Ng | [Github](https://github.com/joelngyx) | [Portfolio](team/joel.md)
-![Kingston Kuan Jun Xiang](https://avatars0.githubusercontent.com/u/35717847?s=460&u=fd4a6977a95813022f82662ada8694105196b9a0&v=4) | Kingston Kuan Jun Xiang | [Github](https://github.com/kstonekuan) | [Portfolio](team/kstonekuan.md)
+![Joseph Abraham](https://avatars2.githubusercontent.com/u/60458098?s=400&u=b9f8cc1b44e9c8e03d8d9fe83d74338113c3a7f8&v=4) | Joseph Abraham | [Github](https://github.com/josephhhhhhhhh) | [Portfolio](docs/team/joseph.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Lim An Qi | [Github](https://github.com/anqi20) | [Portfolio](docs/team/anqi.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Joel Ng | [Github](https://github.com/joelngyx) | [Portfolio](docs/team/joel.md)
+![Kingston Kuan Jun Xiang](https://avatars0.githubusercontent.com/u/35717847?s=460&u=fd4a6977a95813022f82662ada8694105196b9a0&v=4) | Kingston Kuan Jun Xiang | [Github](https://github.com/kstonekuan) | [Portfolio](docs/team/kstonekuan.md)
```
Comment (kstonekuan) [inline docs/AboutUs.md]: Sorry just need to change this back to without the docs and need to change your own file name to be your github username

**Review by kstonekuan — CHANGES_REQUESTED** [review]

File: docs/team/joseph.md [hunk 3]
```diff
@@ -1,6 +1,33 @@
-# John Doe - Project Portfolio Page
+# Joseph Abraham - Project Portfolio Page
```
Comment (Zhi-You) [inline docs/team/joseph.md]: Naming of the file should be that of your github username. After changing that i think it will be good to be merged!

**Review by kstonekuan — APPROVED** [review]

File: docs/team/joseph.md (hunk 3 above)
Comment (josephhhhhhhhh) [reply docs/team/joseph.md]: Roger, thank you!

File: docs/AboutUs.md (hunk 2 above)
Comment (josephhhhhhhhh) [reply docs/AboutUs.md]: Gotcha

File: docs/team/joseph.md (hunk 1 above)
Comment (josephhhhhhhhh) [reply docs/team/joseph.md]: Oh yeah you're right, will make adjustments to it in a future update! Thanks for pointing it out!

**Pull request merged**